src/io/flutter/inspector/EvalOnDartLibrary.java

```diff
@@ -32,46 +33,35 @@
 public class EvalOnDartLibrary implements Disposable {
   private String isolateId;
   private final VmService vmService;
+  @SuppressWarnings("FieldCanBeLocal") private final PerfService perfService;
```

Comment:
why the suppress warnings annotation? The field appears to be final.

Reply:
It wants me to remove this field and instead just use a local in the constructor. I'd prefer to keep it a field so it's available for other uses :)